docs/device-management/5-rpc/README.md

```diff
@@ -0,0 +1,27 @@
+# Remote Procedure Call (RPC)
+
+## What is a Remote Procedure Call?
+
+A Remote procedure call (RPC) allows you to run a command on an IoT device in
+the field, optionally supplying your own input data. The device will execute the
+command, returning state information and output data (if any).
+
+RPCs are synchronous and run on the device as soon as they are called/received.
+Devices only respond to the RPC methods to which they have registered with the
+Golioth server. An RPC may be initiated from the Golioth web console, REST API,
+or using command line tools.
+
+### Use Cases
+
+Here are some ideas on what can be accomplished with Golioth's RPC feature.
+
+- Device reboot
+  - It sounds simple, but the ability to remotely reboot your device is a
+    surprisingly handy.
+- Set a different logging level for one device
+  - When a device in the field is misbehaving, use an RPC to remotely enable
+    debug logs.
```

Comment:
Not necessarily opposed to this, but I think we would typically suggest a settings override for this type of operation. Perhaps fine to mention as something you _could_ do though.